tests/test_deserialization.py

```diff
@@ -51,9 +51,19 @@ class TestCase:
             ],
         ),
         TestCase(
-            "tests/record_issue_15.json",
+            "tests/issue_15.json",
             ['{job="integrations/azure-logexport"}'],
         ),
+        TestCase(
+            "tests/issue_19_sample_1.json",
+            ['{job="integrations/azure-logexport"}'],
```

Comment:
I think we should add a type here. This sample is an [alert](https://learn.microsoft.com/en-us/rest/api/defenderforcloud-composite/alerts/list?view=rest-defenderforcloud-composite-latest&tabs=HTTP#alert) if I'm not mistaken. Should we set the `type` label to `type=Alert/<AlertType>` then?

Reply:
I'm not sure if they sent non-alerts.